metis-indexing/src/main/java/eu/europeana/indexing/IndexerImpl.java

```diff
@@ -43,6 +34,7 @@ public class IndexerImpl implements Indexer {
 
   private final IndexingSupplier<StringToFullBeanConverter> stringToRdfConverterSupplier;
 
+  private final TombstoneUtil tombstoneUtil = new TombstoneUtil();
```

Comment:
use methods from static class? what do you think? I am seeing this from the perspective every indexer will be needed to create this object, than using one static class is just one time on the ram.